static_src/test/unit/stores/service_instance_store.spec.js

```diff
@@ -603,4 +604,45 @@ describe('ServiceInstanceStore', function() {
       expect(spy).toHaveBeenCalledOnce();
     });
   });
+
+  describe('on instance error', function() {
+    const testCFError = {
+      code: 40023,
+      description: 'Cannot bind',
+      error_code: 'CF-BindingCannot'
+    };
+
+    it('should toggle the error value of the instance', function() {
+      const instanceGuid = 'zvxcsdf34sint';
+      const instance = { guid: instanceGuid };
+      const err = testCFError;
+
+      ServiceInstanceStore.push(instance);
+
+      serviceActions.instanceError(instanceGuid, err);
+
+      const actual = ServiceInstanceStore.get(instanceGuid);
+      const expected = Object.assign({}, instance, {
+        error: err
+      });
+
+      expect(actual).toEqual(expected);
+    });
+
+    it('should emit change if errored instance found', function() {
+      const instanceGuid = 'zvxcsdf34sint';
+      const instance = { guid: instanceGuid };
+      const err = testCFError;
+
+      ServiceInstanceStore.push(instance);
+
+      const spy = sandbox.spy(ServiceInstanceStore, 'emitChange');
+
+      serviceActions.instanceError('fake-guid', err);
```

Comment:
It's to check that emitChange was only called on one of them, the first valid one, so 2nd call. So `expect(spy).toHaveBeenCalledOnce();`  is the only assertion.